homeassistant/util/temperature.py

```diff
@@ -10,15 +10,19 @@
 def fahrenheit_to_celsius(fahrenheit: float, interval: bool = False) -> float:
     """Convert a temperature in Fahrenheit to Celsius."""
     if interval:
-        return fahrenheit / 1.8
-    return (fahrenheit - 32.0) / 1.8
+        result = fahrenheit / 1.8
+    else:
+        result = (fahrenheit - 32.0) / 1.8
+    return round(result, 1)
```

Comment:
We can't do the rounding in the util. Each integration has to decide what kind of rounding it needs.